src/Typeclasses/Classes/Group.elm

```diff
@@ -0,0 +1,53 @@
+module Typeclasses.Classes.Group exposing
+    ( Group
+    , floatProduct, intProduct, numberSum
+    )
+
+{-| Group typeclass definition and its instances for basic types.
+
+
+# Definition
+
+@docs Group
+
+-}
+
+import Typeclasses.Classes.Monoid exposing (Monoid)
+
+
+{-| Explicit typeclass which implements group operations for type `a`.
+-}
+type alias Group a =
+    { monoid : Monoid a
+    , inverse : a -> a
+    }
+
+
+{-| Construct an instance for any type which satisfies Elm's `number` magic constraint.
+Implements multiplication.
+-}
+floatProduct : Group Float
+floatProduct =
+    { monoid = Typeclasses.Classes.Monoid.numberProduct
+    , inverse = \number -> 1 / number
+    }
+
+
+{-| Construct an instance for any type which satisfies Elm's `number` magic constraint.
```

Comment:
Refers to the `number` constraint instead of `Int`.

Reply:
Will change documentation to Int